compact.go

```diff
@@ -199,6 +198,20 @@ func (c *LeveledCompactor) plan(dms []dirMeta) ([]string, error) {
 	return nil, nil
 }
 
+// excludeFreshMeta removes newest block with the shortest range from given meta files.
+func excludeFreshMeta(dms []dirMeta) []dirMeta {
+	sort.Slice(dms, func(i, j int) bool {
```

Comment:
Why are we not comparing just the MinTimes?

Reply:
Hm.. Yea - you might be right. Wanted to be too explicit here and not miss any edge case. This makes sense.

Reply:
so actually it changes the logic completely - it's not the same. But maybe it is something we want. If we sort only by MinTimes in case:
```
		{
			// We have 20, 20, 20, 60, 60 range blocks.
			metas: []dirMeta{
				metaRange("1", 0, 20, nil),
				metaRange("2", 20, 40, nil),
				metaRange("3", 40, 60, nil),
				metaRange("4", 60, 120, nil),
				metaRange("5", 120, 180, nil),
			},
		},
```
We compact 1, 2, 3. It might be the case that 3 is generated from WAL dump (4 and 5 might be generated from compaction), but we still include 3 in the process. That's wrong.

The question is now, if such case make any sense. Maybe there is no possibility to have such situation.

Reply:
I would actually compare MaxTimes here.. isn't that a true "Last block"?